### PR TITLE
[Doc] Fixing Links and describe more details

### LINKED ISSUE #15: Link broken
In your README.md file, there is a broken link:  https://github.com/SAP/devops-docker-images/blob/master/mta-archive-builder

it's in the DockerFiles list.

### PR DESCRIPTION
**Pull request by radsoulbeard**
fixes #15 
Fixes the links and pointing them directly to the image.
Adding another section for Dockerfiles not published as images.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,17 +10,27 @@ This repository contains Dockerfiles that are designed to run project "Piper" pi
...
+The following images are already published:
 
 | Name | Description | Link |
 |------|-------------|------|
-| Jenkins | Preconfigured Jenkins to run project "Piper" pipelines. | [jenkins-master/](jenkins-master/) |
-| MTA Archive Builder | Build SAP Multitarget Applications with the [MTA archive builder](https://help.sap.com/viewer/58746c584026430a890170ac4d87d03b/Cloud/en-US/ba7dd5a47b7a4858a652d15f9673c28d.html). | [mta-archive-builder/](mta-archive-builder/) |
-| CM Client | Interact with SAP Solution Manager or CTS using the command line. | [cm-client/](cm-client/) |
-| CloudFoundry CLI | Command-line tools for CloudFoundry, with plugins for blue-green deploy and MTA. | [cf-cli/](cf-cli/) |
-| Jenkinsfile Runner| [Jenkinsfile Runner](https://github.com/jenkinsci/jenkinsfile-runner) based on `ppiper/jenkins-master`, allows running a `Jenkinsfile` without a long-running, stateful Jenkins master. | [jenkinsfile-runner/](jenkinsfile-runner/) |
+| Jenkins | Preconfigured Jenkins to run project "Piper" pipelines. | [jenkins-master](https://hub.docker.com/r/ppiper/jenkins-master) |
```
[the remark this answers is not in the document]
Comment (1084565) [reply README.md]: +1

File: README.md [hunk 2]
```diff
@@ -10,17 +10,27 @@ This repository contains Dockerfiles that are designed to run project "Piper" pi
...
+| MTA Archive Builder | Build SAP Multitarget Applications with the [MTA archive builder](https://help.sap.com/viewer/58746c584026430a890170ac4d87d03b/Cloud/en-US/ba7dd5a47b7a4858a652d15f9673c28d.html). | [mta-archive-builder](https://hub.docker.com/r/ppiper/mta-archive-builder) |
+| CM Client | Interact with SAP Solution Manager or CTS using the command line. | [cm-client](https://hub.docker.com/r/ppiper/cm-client) |
+| CloudFoundry CLI | Command-line tools for CloudFoundry, with plugins for blue-green deploy and MTA. | [cf-cli](https://hub.docker.com/r/ppiper/cf-cli) |
+| Jenkinsfile Runner| [Jenkinsfile Runner](https://github.com/jenkinsci/jenkinsfile-runner) based on `ppiper/jenkins-master`, allows running a `Jenkinsfile` without a long-running, stateful Jenkins master. | [jenkinsfile-runner](https://hub.docker.com/r/ppiper/jenkinsfile-runner) |
```
[the remark this answers is not in the document]
Comment (radsoulbeard) [reply README.md]: Not every image has its own Readme so far, but a dockerhub repo

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (radsoulbeard) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -10,17 +10,27 @@ This repository contains Dockerfiles that are designed to run project "Piper" pi
 
 For detailed usage information please check the README.md in the corresponding folder.
 
-### Dockerfiles
+### Docker Images
 
-The following files are still being prepared, and are not yet released:
+The following images are already published:
 
 | Name | Description | Link |
```
[the remark this answers is not in the document]
Comment (radsoulbeard) [reply README.md]: done

File: README.md [hunk 4]
```diff
@@ -10,17 +10,27 @@ This repository contains Dockerfiles that are designed to run project "Piper" pi
...
-| CM Client | Interact with SAP Solution Manager or CTS using the command line. | [cm-client/](cm-client/) |
-| CloudFoundry CLI | Command-line tools for CloudFoundry, with plugins for blue-green deploy and MTA. | [cf-cli/](cf-cli/) |
-| Jenkinsfile Runner| [Jenkinsfile Runner](https://github.com/jenkinsci/jenkinsfile-runner) based on `ppiper/jenkins-master`, allows running a `Jenkinsfile` without a long-running, stateful Jenkins master. | [jenkinsfile-runner/](jenkinsfile-runner/) |
+| Jenkins | Preconfigured Jenkins to run project "Piper" pipelines. | [jenkins-master](https://hub.docker.com/r/ppiper/jenkins-master) |
+| MTA Archive Builder | Build SAP Multitarget Applications with the [MTA archive builder](https://help.sap.com/viewer/58746c584026430a890170ac4d87d03b/Cloud/en-US/ba7dd5a47b7a4858a652d15f9673c28d.html). | [mta-archive-builder](https://hub.docker.com/r/ppiper/mta-archive-builder) |
+| CM Client | Interact with SAP Solution Manager or CTS using the command line. | [cm-client](https://hub.docker.com/r/ppiper/cm-client) |
```
[the remark this answers is not in the document]
Comment (radsoulbeard) [reply README.md]: done

**Pull request merged**